framework/BaseYii.php

```diff
@@ -128,7 +128,7 @@ public static function getVersion()
      */
     public static function getAlias($alias, $throwException = true)
     {
-        if (strncmp($alias, '@', 1)) {
+        if (!self::isAlias($alias)) {
```

Comment:
This method return 3 value.
First original parameter.
To alias path.
Three value of of alias not found false